static/templates/about_zulip.hbs

```diff
@@ -8,12 +8,16 @@
             <p>
                 <strong>Zulip Server</strong>
                 <br />
-                Version {{zulip_version}}
-                <i class="fa fa-copy tippy-zulip-tooltip" data-tippy-content="Copy version" data-tippy-placement="right" data-clipboard-text="{{zulip_version}}"></i>
+                {{#tr}}
+                Version {zulip_version}
+                {{/tr}}
+                <i class="fa fa-copy tippy-zulip-tooltip" data-tippy-content="{{t 'Copy version' }}" data-tippy-placement="right" data-clipboard-text="{{zulip_version}}"></i>
                 {{#if is_fork}}
-                <br />
-                Forked from upstream at {{zulip_merge_base}}
-                <i class="fa fa-copy tippy-zulip-tooltip" data-tippy-content="Copy version" data-tippy-placement="right" data-clipboard-text="{{zulip_merge_base}}"></i>
+                    <br />
+                    {{#tr}}
+                    Forked from upstream at {zulip_merge_base}
```

Comment:
I'm not sure, is this necessary to translate?

Reply:
I'm not sure it's necessary, but it's probably simpler to not make an exception.